tests/sc_validate_table_name.test/t1_illegal_characters.expected

```diff
@@ -0,0 +1 @@
+[create table "table-with>illegal<chars" (i int)] failed with rc -3 Table name contains illegal characters. name='table-with>illegal<chars'
```

Comment:
This is similar to [sql.test/t16_01.req](https://github.com/morgando/comdb2/blob/main/tests/sql.test/t16_01.req). I think you might need to update the expected output for that test since the error message changed (or you could remove the table name tests from sql.test/t16_01.req so that they're all here).